src/pages/community/index.js

```diff
@@ -169,12 +169,14 @@ export default ({ ...data }) => {
                             around the globe.
                         </P>
                         <ProjectActions>
-                            <Button to="#" secondary>
+                            <Button to="/article" primary>
                                 Learn how they did it
                             </Button>
                             <ProjectSignUpForm
                                 triggerComponent={
-                                    <Link tertiary>Share your project</Link>
+                                    <Button secondary>
```

Comment:
We might want some spacing between these buttons?
![Screen Shot 2020-02-27 at 3 10 48 PM](https://user-images.githubusercontent.com/9064401/75482709-5a574000-5973-11ea-9f4c-19c59f07bcb1.png)